src/chrome/content/rules/Heise.de.xml

```diff
@@ -39,74 +39,58 @@
-			<test url="http://www.heise.de/foto/"/>
 			<test url="http://www.heise.de/js/foto/galerie/angularjs/partials/ngDialog.inc"/>
 			<test url="http://www.heise.de/js/foto/galerie/angularjs/partials/photo/diashow.html"/>
 			<test url="http://www.heise.de/js/ho/jwplayer-6.10/skin/bekle.xml" />
-			<test url="http://www.heise.de/mediadaten/online/" />
-			<test url="http://www.heise.de/news-extern/news.html" />
-			<test url="http://www.heise.de/newsletter/manage/heisec-summary" />
-			<test url="http://www.heise.de/newsticker/" />
-			<test url="http://www.heise.de/security/artikel/" />
-			<test url="http://www.heise.de/security/hilfe/" />
-			<test url="http://www.heise.de/security/news/" />
-			<test url="http://www.heise.de/security/news/archiv/" />
-			<test url="http://www.heise.de/security/news/news-atom.xml" />
-			<test url="http://www.heise.de/security/tools/" />
 
 			<!--	-ve:
 					-->
-			<test url="http://heise.de/avw-bin/ivw/CP/barfoo/ho/2793053/0.gif" />
+			<test url="http://www.heise.de/avw-bin/ivw/CP/barfoo/ho/2793053/0.gif" />
+			<test url="http://www.heise.de/ivw-bin/ivw/CP/download" />
 			<test url="http://www.heise.de/favicon.ico" />
 			<test url="http://www.heise.de/foto/icons/galerie/avatar_48.png" />
 			<test url="http://www.heise.de/icons/ho/heise_online_logo_top.gif" />
 			<test url="http://www.heise.de/imgs/02/1/2/5/5/4/2/0/newsletter_briefumschlag_ho_klein2-4b495d4dbd4ddf57.png" />
 			<test url="http://www.heise.de/ix/images/navigation_arrow.png" />
+			<test url="http://www.heise.de/scale/geometry/250/q50/imgs/18/1/7/6/3/3/3/9/heise-tls-0d8d7711ff4ce298.png" />
 			<test url="http://www.heise.de/js/heise.min.js" />
 			<test url="http://www.heise.de/js/ho/login.min.js" />
-			<test url="http://www.heise.de/security/dienste/portscan/test/go.shtml" />
-			<test url="http://www.heise.de/security/icons/bg_blue.gif" />
+			<test url="http://www.heise.de/security" />
+			<test url="http://www.heise.de/security/" />
 			<test url="http://www.heise.de/stil/heise-ui.css" />
 			<test url="http://www.heise.de/support/lib/teaser_linking.js" />
+			<test url="http://www.heise.de/software/screenshots/" />
+
+
 
 	<rule from="^http://m\.heise\.de/(js/|icons/|[ai]vw-bin/|security/icons/|stil/|fonts/)" to="https://m.heise.de/$1"/>
+		<exclusion pattern="^http://m\.heise\.de/$" />
 
-	<rule from="^http://heise\.de/"
-		to="https://www.heise.de/"/>
+	<rule from="^http://(abo|piwik\.shop|prophet|shop)\.heise\.de/" to="https://$1.heise.de/"/>
 
 	<rule from="^http://www\.shop\.heise\.de/"
 		to="https://shop.heise.de/" />
 
-	<rule from="^http:"
-		to="https:" />
+	<rule from="^http://www\.heise\.de/security"
+		to="https://www.heise.de/security/" />
+
+	<rule from="^http://www\.heise\.de/([ai]vw-bin/|favicon\.ico|foto/icons/|icons/|ix/images/|imgs/|js/|scale/|security/|software/screenshots/|stil/|support/lib/)" to="https://www.heise.de/$1"/>
+		<exclusion pattern="^http://www\.heise\.de/$" />
```

Comment:
The above exclusion is not needed.

For the below exclusions it would be nice, if you can find the issue with them to document. Referring to git log you added them yourself a year ago.